Sources/Logger/Logging.swift

```diff
@@ -8,6 +8,7 @@
 import Foundation
 
 public protocol Logging {
+    var id: UUID { get }
```

Comment:
Celkove nevidim prinos id pro loggery. Je to tak ze, aplikaci třeba pracovat s konkretni instanci loggeru, kterou si sama vytvorila, ale proste se vám nechce si pamatovat referenci, tak to taháte zpětné pomoci uuid?